docker/build-tool.sh

```diff
@@ -117,15 +117,6 @@ function push() {
     $DOCKER_BINARY push "${REPOSITORY}/leonardo-${IMAGE}:${DOCKER_TAG}"
     $DOCKER_BINARY tag "${REPOSITORY}/leonardo-${IMAGE}:${DOCKER_TAG}" "${REPOSITORY}/leonardo-${IMAGE}:${GIT_BRANCH}"
     $DOCKER_BINARY push "${REPOSITORY}/leonardo-${IMAGE}:${GIT_BRANCH}"
-
-    # For backwards compatibility; also tag and push jupyter images named 'leonardo-notebooks'.
```

Comment:
Means https://github.com/broadinstitute/firecloud-develop/pull/1481 should be merged first, right?

Reply:
yes will merge that ahead of this